buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -148,6 +149,7 @@ object Dependencies {
     const val gradleNexusPublishPlugin = "io.github.gradle-nexus:publish-plugin:${Versions.GRADLE_NEXUS_PUBLISH_PLUGIN}"
     const val gradleVersionsPlugin = "com.github.ben-manes:gradle-versions-plugin:${Versions.GRADLE_VERSIONS_PLUGIN}"
     const val gson = "com.google.code.gson:gson:${Versions.GSON}"
+    const val icu4j = "com.ibm.icu:icu4j:${Versions.ICU4J}"
```

Comment:
Can't we avoid adding this library? It will increase our SDK size a lot

Reply:
The only way I see would be to use Transliterator from android OS. But this is only available prior to API 29 =|

Reply:
Perhaps we could include transliteration support in a different module?